NuGet.config

```diff
@@ -8,7 +8,6 @@
     <clear />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="dotnet-coreclr" value="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" />
-    <!-- TODO: Remove when an updated coverlet.console package is pushed to nuget.org. Issue: #39595 -->
```

Comment:
👿👿👿

Reply:
Hmmm the bot removed the comment:

https://github.com/dotnet/corefx/commit/4f9928fcae3d1e5d809e919a704ea04905842987

I wonder why? @riarenas